FreshRSS/FreshRSS
# Deleted: bookmark.svg

Before:
Pafat theme: a white star
![grafik](https://user-images.githubusercontent.com/1645099/204063349-de747aad-ae27-492f-a985-1ad59ef86e95.png)

After:
![grafik](https://user-images.githubusercontent.com/1645099/204063391-96282cb3-0675-4434-b446-2a2432045afa.png)


The same star as the favorite icon. It keeps a white star in the left navigation bar
![grafik](https://user-images.githubusercontent.com/1645099/204063386-806518d5-4852-47c9-93f4-bf78b0c4b2d1.png)


Changes proposed in this pull request:

- deleted bookmark.svg (it was mostly the same icon like starred.svg)
- Pafat theme: all.svg is now in grey. CSS sets the filter to white


How to test the feature manually:

1. check the stars in all themes, especially in Pafat theme


Pull request checklist:
